cedar-lean/Cedar/Partial/Response.lean

```diff
@@ -73,10 +75,6 @@ structure Response where
     Does include policies that are definitely satisfied (residual `true`).
   -/
   residuals : List Residual
-  /--
```

Comment:
I really like this change. Removing the dangling request makes me want to port it to the Rust impl